migrations/2023-10-04-120026_add_payment_type_column_in_payment_intent/up.sql

```diff
@@ -0,0 +1,11 @@
+-- Your SQL goes here
+
+CREATE TYPE "PaymentType" AS ENUM (
+    'normal',
+    'new_mandate',
+    'setup_mandate',
+    'recurring_mandate'
+);
+
+ALTER TABLE payment_intent
+ADD COLUMN payment_type "PaymentType" NOT NULL DEFAULT 'normal';
```

Comment:
This will mark all the previous mandate / setup mandate transactions as 'normal' payments. Is that okay?

Reply:
Ah, good catch. We probably don't want to do that. I'll update it.